src/main/java/org/myrobotlab/service/Runtime.java

```diff
@@ -1476,7 +1474,7 @@ synchronized public static void unregister(String inName) {
       }
 
       if (updatedServiceLists) {
-        runtime.broadcast("publishInterfaceToPossibleServices");
+        runtime.invoke("publishInterfaceToPossibleServices");

```

Comment:
this isn't part of the service lifecycle publisher / listener interfaces.. if we are putting it here.. it should be defined in the interface also.

Reply:
it shouldn't be either - its currently an very optimized method that only is useful for the ui